src/main/java/com/ericsson/eiffelcommons/http/HttpRequest.java

```diff
@@ -331,10 +331,12 @@ private URIBuilder addParametersToURIBuilder(URIBuilder builder) {
      * @throws URISyntaxException
      */
     private URIBuilder createURIBuilder() throws URISyntaxException {
-        if (endpoint.startsWith("/")) {
+        if (endpoint != null && !endpoint.isEmpty() && endpoint.startsWith("/")) {
```

Comment:
Change to StrungUtils.isEmpty

Reply:
I had that earlier and travis couldn't find the package. I can try later again.

Reply:
Travis had some temporary problems it seems. StringUtils now seems to work.